L/Libxc/Libxc_GPU/build_tarballs.jl

```diff
@@ -5,61 +5,53 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "Libxc_GPU"
-version = v"6.1.0"
+version = v"7.0.0"
 include("../sources.jl")
 
-sources = [
-    sources;
-    DirectorySource("./bundled")
-]
-
 # Bash recipe for building GPU version
 # Notes:
 #   - 3rd and 4th derivatives (KXC, LXC) not built since gives a binary size of ~200MB
 script = raw"""
-cd $WORKSPACE/srcdir/libxc-*/
 
-# Needed for Libxc 6.1.0 as these backport some fixes on libxc master
-# On Libxc > 6.1.0 we can also remove the -DBUILD_TESTING=OFF
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/cmake-cuda.patch
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/source-fixes.patch
+cd $WORKSPACE/srcdir/libxc-*/
 
 mkdir libxc_build
 cd libxc_build
-cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+
+mv ${WORKSPACE}/destdir/cuda/lib ${WORKSPACE}/destdir/cuda/lib64
```

Comment:
Doesn't this leave leftovers in the final tarballs?  Why do you need to move it?

Reply:
CMake looks for the cuda installation in `cuda/lib64` rather than `cuda/lib`. Other CUDA packages either do this, or they create a symlink. I'm happy to change for the latter, if it is indeed cleaner.

Reply:
The point is that the tarball has a `cuda` directory full of empty subdirectories.  That shouldn't exist at all.

Reply:
Cleaned-up. It looks like the offending line was `skip_audit=true, dont_dlopen=true` in the arguments of the `build_tarball()` function. I am not sure what it did, but it does not seem necessary. I successfully tested the new build as well.